patches/@lerna+conventional-commits+5.6.2.patch

```diff
@@ -1,20 +0,0 @@
-diff --git a/node_modules/@lerna/conventional-commits/lib/recommend-version.js b/node_modules/@lerna/conventional-commits/lib/recommend-version.js
-index a982adf..453ef14 100644
---- a/node_modules/@lerna/conventional-commits/lib/recommend-version.js
-+++ b/node_modules/@lerna/conventional-commits/lib/recommend-version.js
-@@ -58,6 +58,15 @@ function recommendVersion(pkg, type, { changelogPreset, rootPath, tagPrefix, pre
-         // we still need to bump _something_ because lerna saw a change here
-         let releaseType = data.releaseType || "patch";
- 
-+        // Don't gratuitously break compatibility with clients using `^0.x.y`.
-+        if (semver.major(pkg.version) === 0) {
-+          if (releaseType === "major") {
-+            releaseType = "minor";
-+          } else if (releaseType === "minor") {
-+            releaseType = "patch";
-+          }
-+        }
-+
```

Comment:
Is this change no longer necessary?

Reply:
correct, it's one of the upstream fixes I mentioned https://github.com/lerna/lerna/blob/28c8ef232d29085ce9a9ba2bff96fae9017b3963/libs/core/src/lib/conventional-commits/recommend-version.ts#L76-L81

Reply:
That check is only applicable for non pre-release, which is the problem we had with lerna 5, See https://github.com/Agoric/agoric-sdk/issues/8242

Reply:
I elected to ramp all packages into non-zero major versions rather than maintain the Lerna patch going forward.

Reply:
Isn't the problem that lerna will elect to use a new 1.x version for pre-release but when cutting the final release it will keep using a 0.x version?